### PR TITLE
Added support for mirroring documents.

### PR DESCRIPTION
**Pull request by patriksvensson**
Tested and works on my machine, but would feel better if someone else tried this out before merging.

### CODE REVIEW

**Comment by patriksvensson** [comment]
Should take care of issue #92 submitted by @endrton.

File: src/Cake.Web/Cake.Web.csproj [hunk 1]
```diff
@@ -267,7 +267,53 @@
     <Content Include="Views\Blog\Category.cshtml" />
     <Content Include="Views\Addin\Index.cshtml" />
     <Content Include="Views\Shared\_AddinWarning.cshtml" />
-    <Content Include="App_Data\**\*.md" />
+    <Content Include="App_Data\blog\2015-07-05-cake-v0.4.3-released.md" />
```
Comment (patriksvensson) [inline src/Cake.Web/Cake.Web.csproj]: Hmm, not sure what happened here. Strange.

Comment (gep13) [inline src/Cake.Web/Cake.Web.csproj]: @patriksvensson looks like Visual Studio trying to be "clever". Do you want to put this back the way it was, or do you want them explicitly added into the csproj file?

Comment (devlead) [inline src/Cake.Web/Cake.Web.csproj]: @gep13 @patriksvensson  We don't want to add each explicitly, so that part needs to be reverted.

**Comment by patriksvensson** [comment]
@gep13 This should be fixed now.

**Comment by gep13** [comment]
@patriksvensson this looks good to me!

**Pull request merged**